index.html

```diff
@@ -2342,6 +2333,50 @@
           </li>
         </ol>
       </section>
+      <section>
+        <h2>
+          Filter payment options algorithm
+        </h2>
+        <p>
+          The <dfn data-lt="filter payment options">filter payment options
+          algorithm</dfn> runs when user agent presents the user with the
+          collection of payment options available for selection. It MUST run the
+          following steps:
+        </p>
+        <ol>
+          <li>Let <var>matchingOptions</var> be an empty list.</li>
+          <li>Let <var>request</var> be the <a>PaymentRequest</a> object that
+          the user is interacting with.
+          <li>For each <var>requestMethod</var> in
+            <var>request</var>.<a>[[\serializedMethodData]]</a>:
+            <ol type="I">
+              <li>Let <var>filters</var> be <var>requestMethod.data</var>
+                members whose values are lists of strings.</li>
+              <li>For each <var>option</var> of the installed
+                <a>payment apps</a>:
+                <ol type="i">
+                  <li>Let <var>optionMatch</var> be true.</li>
+                  <li>For <var>filterName</var> in the keys of
+                    <var>requestMethod</var>:
+                    <ol type="a">
+                      <li>If <var>option.capabilities</var> does not have key
```

Comment:
Where is option.capabilities defined? I thought a payment app was an abstract concept, not an object which had properties.